### PR TITLE
chore(pr-template): add reviewer checklist section

### PR DESCRIPTION
**Pull request by ArmandDu**
## Describe the feature

Add new Reviewer Checklist in Pull Request Template

## Describe the context for the feature

We want to have consistent quality for the code review. The solution was to add a checklist.
The reviewer(s) don't have to check all boxes but at least make sure that they thought about it.

The goal is to improve review quality.

### CODE REVIEW

**Review by devegili1 — APPROVED** [review]

**Comment by devegili1** [comment]
image

File: pull_request_template.md [hunk 1]
```diff
@@ -24,15 +24,31 @@ Please describe any affects that this might have on other parts of the applicati
 
 Please describe in detail how you tested your changes using checkboxes, so the reviewer can check of what you have tested as they also test the bug fix.
 
--   [ ] xxx
--   [ ] yyy
+- [ ] xxx
```
Comment (iverds) [inline pull_request_template.md:27]: Not directly relevant, but think we should have some suggestions here

Comment (iverds) [reply pull_request_template.md:27]: Or remove it

Comment (ArmandDu) [reply pull_request_template.md:27]: I can make a new post on slack for that if you want and/or discuss it during the next meeting/tech corner

**Pull request merged**